src/solvers/dgmulti/sbp.jl

```diff
@@ -302,7 +302,9 @@ Constructs a single-element [`VertexMappedMesh`](@ref) for a single periodic ele
 a DGMulti with `approximation_type` set to a periodic (finite difference) SBP operator from
 SummationByPartsOperators.jl.
 """
-function CartesianMesh(dg::DGMultiPeriodicFDSBP{NDIMS}) where {NDIMS}
+function CartesianMesh(dg::DGMultiPeriodicFDSBP{NDIMS};
+                       coordinates_min=ntuple(_ -> -1.0, NDIMS),
+                       coordinates_max=ntuple(_ -> 1.0, NDIMS)) where {NDIMS}
```

Comment:
Type stability: Although we don't have this feature in its full form, I would like to let the real type of the mesh depend on the chosen real type of the solver here. Thus, we should use something like
```suggestion
function CartesianMesh(dg::DGMultiPeriodicFDSBP{NDIMS};
                       coordinates_min=ntuple(_ -> -one(real(dg)), NDIMS),
                       coordinates_max=ntuple(_ -> one(real(dg)), NDIMS)) where {NDIMS}
```